README.md

```diff
@@ -35,6 +35,23 @@ To release a new version with your changes, run `yarn changeset add` and follow
 
 When your PR is merged, changeset will analyse the changes and create a PR to release the new version.
 
+For an efficient and secure process, a script is available to update the version of commercial lib in Frontend to your specified version. You can find the script [here](./bump_commercial.sh).
```

Comment:
I think this should be in its own section and made a bit more obvious that the command is to be run in consumers i.e. frontend/DCR?